src/plugins/ui/edition.ts

```diff
@@ -639,9 +639,10 @@ export class EditionPlugin extends UIPlugin {
    */
   getReferencedRanges(): Range[] {
     const editionSheetId = this.getters.getEditionSheet();
-    return this.currentTokens
+    const referenceRanges = this.currentTokens
       .filter((token) => token.type === "REFERENCE")
       .map((token) => this.getters.getRangeFromSheetXC(editionSheetId, token.value));
+    return referenceRanges.filter((range) => !range.invalidSheetName && !range.invalidXc);
```

Comment:
Chaining the `map` and the `filter` would look cooler than an intermediary variable 😎